DOCS/man/ao.rst

```diff
@@ -149,13 +149,8 @@ Available audio output drivers are:
 
     The following global options are supported by this audio output:
 
-    ``--pipewire-buffer-samples=<n>``
-        Set the audio buffer size in samples. A higher value buffers
-        more data, and has a lower probability of buffer underruns. A smaller
-        value makes the audio stream react faster, e.g. to playback speed
-        changes.
-        The effective size of the buffer depends on the amount of buffered
-        samples and the samplerate.
+    ``--pipewire-buffer=<1-2000|native>``
+        The same as ``--pulseaudio-buffer``.
```

Comment:
I would copy the text so avoid depending on documentation that might theoretically go away.